.travis.yml

```diff
@@ -2,24 +2,31 @@ notifications:
   email: false
 
 install:
-  - pip install .
   - python setup.py build_ext --inplace
+  - pip3 install .
 script:
-  - pip install pytest~=3.6.1
-  - pytest
+  - pip3 install nose2 nose2[coverage_plugin]>=0.6.5
+  - nose2 --with-coverage
```

Comment:
I would have a preference to keep using pytest, for consistency with the other TVB builds.
Anyway, nose does not solve the seg fault issue.

Reply:
Yes, it appears to be a pytest issue.

Reply:
why you say it is pytest? 
the build are still failing, and tests are not really execute in the latest Travis run.... am I missing something?

Reply:
https://github.com/enthought/comtypes/issues/204 I got this same error. The current failures are due to something else. I am still investigating and will report you back if nose actually fixes the issue.